onnxruntime/core/providers/cpu/tensor/slice.cc

```diff
@@ -260,19 +260,14 @@ Status SliceBase::PrepareForCompute(const std::vector<int64_t>& raw_starts,
 }
 
 // Slice V10 & DynamicSlice
-void SliceBase::FillVectorsFromInput(const OpKernelContext* context,
+void SliceBase::FillVectorsFromInput(const Tensor* start_tensor,
+                                     const Tensor* ends_tensor,
```

Comment:
starts_tensor and ends_tensor are required so references please. axes_tensor and steps_tensor are optional so need to be pointers.